src/zulip.rs

```diff
@@ -291,8 +291,9 @@ async fn workqueue_commands(
                 None => String::from("Not set (i.e. unlimited)"),
             };
 
-            let mut response = format!("Assigned rust-lang/rust PRs: {prs}\n");
-            writeln!(response, "Review capacity: {capacity}")?;
+            let mut response = format!("`rust-lang/rust` PRs in your review queue: {prs}\n");
+            writeln!(response, "Review capacity: {capacity}\n")?;
```

Comment:
It might be also useful to see the current number of PR in the queue. *Just a reflection, nothing to act up-on.*

Reply:
Yeah I was also thinking that it would be nice to see the count, added it.